Cargo.toml

```diff
@@ -30,6 +30,6 @@ unsigned-varint = { version = "0.7.1", features = ["futures", "std"] }
 async-std = { version = "1.10.0", features = ["attributes"] }
 env_logger = "0.9.0"
 libipld = { version = "0.14.0", default-features = false, features = ["dag-cbor"] }
-libp2p = "0.48.0"
+libp2p = { version = "0.49.0", features = ["full"] }
```

Comment:
While this is a fair representation of the previous state of things, it makes me wonder which features the tests actually do need — probably not a lot. Would you be up to trying it out?

Reply:
Went on and updated it to use the features the test would need, which are tcp-async-io, noise, and yamux.